GiniVision/Classes/Core/MultipageReviewBottomCollectionCell.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  MultipageReviewBottomCollectionCell.swift
+//  GiniVision
+//
+//  Created by Enrique del Pozo Gómez on 2/1/18.
+//
+
+import Foundation
+
+final class MultipageReviewBottomCollectionCell: UICollectionViewCell {
+    
+    static let identifier = "MultipageReviewBottomCollectionCellIdentifier"
+    static let size = CGSize(width: 107,
+                             height: 192 +
+                                MultipageReviewBottomCollectionCell.shadowHeight +
+                                MultipageReviewBottomCollectionCell.shadowRadius)
+    static let shadowHeight: CGFloat = 2
+    static let shadowRadius: CGFloat = 1
+    let pageIndicatorCircleSize = CGSize(width: 25, height: 25)
+    
+    lazy var roundMask: UIView = {
+        let view = UIView(frame: .zero)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.layer.cornerRadius = 5.0
+        view.layer.masksToBounds = true
+        return view
+    }()
+    
+    lazy var documentImage: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.backgroundColor = Colors.Gini.veryLightGray
+        imageView.contentMode = .scaleAspectFit
+        imageView.clipsToBounds = true
+        return imageView
+    }()
+    
+    lazy var draggableIcon: UIImageView = {
+        let image = UIImage(named: "draggablePageIcon", in: Bundle(for: GiniVision.self), compatibleWith: nil)
+        let imageView = UIImageView(image: image)
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.contentMode = .scaleAspectFill
+        return imageView
+    }()
+    
+    lazy var bottomContainer: UIView = {
+        let view = UIView(frame: .zero)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .white
+        return view
+    }()
+    
+    lazy var pageIndicator: UILabel = {
```

Comment:
The page indicator is a Label? I think the name is misleading - makes me think it's a page indicator. Maybe `pageIndicatorLabel`?